bin/deploy-vhost

```diff
@@ -187,6 +187,9 @@ if ($conf->{redirects_only}) {
 
 # Make database config settings
 if (defined $conf->{'databases'}) {
+    # Empty arrays for the structured configurations
+    $conf->{struct_database_configs}{internal} = ();
```

Comment:
This will set that variable to undef, not an empty list as I think you want, you want `[]` to create an array ref.